modules/pulse_async/pulse.c

```diff
@@ -72,8 +73,8 @@ static void qh(int id, void *data, void *arg)
 	if (pa.paconn)
 		pa.paconn = mem_deref(pa.paconn);
 
-	if (paconn_start(&pa.paconn))
-		tmr_start(&pa.rc, 1000, reconnth, NULL);
+	pa.retry = 0;
+	tmr_start(&pa.rc, 1000, reconnth, NULL);
```

Comment:
Should'nt we try for ever, but increase the delay?
See also: `reconnth()`

Reply:
Add delay increase between 800ms and 2000ms

Reply:
Increasing from 800ms to 2000ms has not much benefit. Tomorrow, we try a good delay and maybe we will retry for ever.

Maybe we should add a `module_event()` that informs the application that there had been `n` retries to connect to pulseaudio.